03-rnaseq/clustering/clustering_heatmap_example.Rmd

```diff
@@ -0,0 +1,421 @@
+
+```{r}
+# Define the file path to the results directory
+data_dir <- "data" # Replace with path to desired data directory
+
+# Create the data folder if it doesn't exist
+if (!dir.exists(data_dir)) {
+  dir.create(data_dir)
+}
+
+# Define the file path to the plots directory
+plots_dir <- "plots" # Can replace with path to desired output plots directory
+
+# Create the plots folder if it doesn't exist
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+```
+
+In the same place you put this `.Rmd` file, you should now have two new empty folders called `data` and `plots`!
+
+## Obtain the dataset from refine.bio
+
+For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-the-data). 
+
+Go to this [dataset's page on refine.bio](https://www.refine.bio/experiments/SRP070849).
+
+Click the "Download Now" button on the right side of this screen.  
+
+<img src="screenshots/download-now.png" width=200>  
+
+Fill out the pop up window with your email and our Terms and Conditions:  
+
+<img src="screenshots/download-email.png" width=500>  
+
+It make take a few minutes for the dataset to process.
+You will get an email when it is ready. 
+
+## About the dataset we are using for this example
+
+For this example analysis, we will use this [acute myeloid leukemia sample dataset](https://alexslemonade.github.io/refinebio-examples/clustering/data/SRP070849.tsv).
+
+The data that we downloaded from refine.bio for this analysis has 19 samples (obtained from 19 mice with acute myeloid leukemia (AML)), containing RNA-sequencing results for types of AML under controlled treatment conditions.
+
+## Place the dataset in your new `data/` folder
+
+Refine.bio will send you a download button in the email when it is ready. 
+Follow the prompt to download a zip file that has a name with a series of letters and numbers and ends in `.zip`. 
+Double clicking should unzip this for you and create a folder of the same name.  
+
+<img src="screenshots/download-folder-structure.png" width=400> 
+
+For more details on the contents of this folder see [these docs on refine.bio](http://docs.refine.bio/en/latest/main_text.html#rna-seq-sample-compendium-download-folder).  
+The `<experiment_accession_id>` folder has the data and metadata TSV files you will need for this example analysis.
+Experiment accession ids usually look something like `GSE1235` or `SRP12345`. 
+
+## Check out our file structure!
+
+Your new analysis folder should contain: 
+
+- The example analysis Rmd you downloaded  
+- A folder called "data" which contains:  
+  - The gene expression  
+  - The metadata TSV  
+
+Your example analysis folder should now look something like this (except with respective experiment accession id and analysis notebook name you are using): 
+
+<img src="screenshots/analysis-folder-structure.png" width=400>
```

Comment:
I am changing these to URLs over on #166 . Sorry this didn't make it into the version you used. I'm trying to rethink parts of the template and how we should go about managing updates.

Reply:
Updated this notebook to include the URLS as you have them in #166 and removed the `screenshots` folder from this module/PR 👍

Reply:
Ah, it's merged in master now, so you probably could have done a merge with master to get that updated. But this works too. :+1:

Reply:
Oh I didn't think that PR updated this particular module but no worries!

Reply:
Oh DuH. You are right.